packages/project/src/db/build.ts

```diff
@@ -201,21 +201,24 @@ const clonePages = async (
 export async function create(
   projectId: Build["projectId"],
   env: "prod",
-  sourceBuild: Build
+  sourceBuild: Build | undefined,
+  client: Prisma.TransactionClient
 ): Promise<void>;
 export async function create(
   projectId: Build["projectId"],
   env: "dev",
-  sourceBuild?: Build
+  sourceBuild: Build | undefined,
+  client: Prisma.TransactionClient
 ): Promise<void>;
 // eslint-disable-next-line func-style
 export async function create(
   projectId: Build["projectId"],
   env: "dev" | "prod",
-  sourceBuild?: Build
+  sourceBuild: Build | undefined,
+  client: Prisma.TransactionClient
```

Comment:
We wanted to use this pattern everywhere:

```
const myDbFunction = 
 (param1: ..., param2: ..., client: Prisma.TransactionClient = prisma) => {
...
}
```

So all db functions can be called either with transaction, or use default `prisma` client